core/carp_pattern.h

```diff
@@ -87,9 +87,7 @@ String Pattern_internal_classend(PatternMatchState *ms, String p) {
             } while (*p != ']');
             return p + 1;
         }
-        default: {
-            return p;
-        }
+        default: { return p; }
```

Comment:
I always have to discard this diff before committing. My `clang-format` version must be different from the one you're using. I'd suggest getting rid of the braces, maybe both versions will behave the same in that case.

Reply:
My version is 7.1.0 BTW.

Reply:
Just pushed it without the braces, can someone confirm older versions format it the same?

Reply:
Actually, my version (10.0.0) does not reformat that line.

Reply:
Oh, right, I'll update my version then.